components/post_view/message_attachments/action_menu/index.ts

```diff
@@ -3,19 +3,31 @@
 
 import {connect, ConnectedProps} from 'react-redux';
 
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
+
 import {PostAction} from '@mattermost/types/integration_actions';
 
 import {GlobalState} from 'types/store';
 import {autocompleteChannels} from 'actions/channel_actions';
 import {autocompleteUsers} from 'actions/user_actions';
 import {selectAttachmentMenuAction} from 'actions/views/posts';
 
+import {ActionFunc, GenericAction} from 'mattermost-redux/types/actions';
+import {Channel, UserProfile} from 'components/suggestion/command_provider/app_command_parser/app_command_parser_dependencies';
+
 import ActionMenu from './action_menu';
 
 export type OwnProps = {
     postId: string;
     action: PostAction;
     disabled?: boolean;
+    actions?: {
+        autocompleteChannels: (term: string, success: (channels: Channel[]) => void, error: () => void) => Promise<void>;
+        autocompleteUsers: (search: string) => Promise<UserProfile[]>;
+        selectAttachmentMenuAction: (postId: any, actionId: any, cookie: any, dataSource: any, text: any, value: any) => (dispatch: any) => Promise<{
+            data: boolean;
+        }>;
+    };
```

Comment:
i dont understand. we are not using `actions` key in component. because of 
```
const mapDispatchToProps = {
    selectAttachmentMenuAction,
    autocompleteChannels,
    autocompleteUsers,
};
```